cmd/lakectl/cmd/local.go

```diff
@@ -20,17 +20,22 @@ const (
 	localDefaultSyncPresign     = true
 	localDefaultMinArgs         = 0
 	localDefaultMaxArgs         = 1
+
+	localPresignFlagName     = "presign"
+	localParallelismFlagName = "parallelism"
+	localGitIgnoreFlagName   = "no-gitignore"
```

Comment:
Not a fan of negative names for flags.  I would ideally like `gitignore` or `process-gitignore` or any appropriate `VERB-gitignore`, default `true`.

Here this is particularly confusing, as the flagname is the reverse of the meaning of the flag.